corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -66,7 +68,17 @@ hqDefine("geospatial/js/case_grouping_map",[
                 return;
             }
 
-            const casesToExport = _.map(self.casesToExport(), function (caseItem) {
+            // Only cases with belonging to groups should be exported
+            let exportableCases = self.casesToExport().filter(function(caseItem) {
```

Comment:
Shouldn't this be a computed property? Does it get refreshed otherwise?

Reply:
This is only calculated once the user export cases / case groups and is only used here. If we changed this to a computable, this will be calculated whenever `casesToExport` changes, which isn't really necessary (and would lighten the load when there are a lot of cases)